modules/customize-certificates-manually-rotate-service-ca.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-signing-certificates.adoc
+
+[id="manually-rotate-service-ca-{context}"]
+= Manually rotate the service CA certificate
+
+The service CA is valid for one year after {product-title} is 
+installed. If you have a cluster that has been running
+for one year without upgrades, then you will need to manually
+refresh the service CA.
+
+This process is automatically performed during an upgrade,
+and should only be followed for clusters that do not perform
```

Comment:
s/upgrade, and should only be followed for/upgrade. Follow these steps only if you do not update your cluster.

Reply:
Updated to use the suggestion.